src/main.ts

```diff
@@ -78,6 +84,8 @@ const keybinds: Keybind[] = [
                 document.querySelector('button[value=add]') as HTMLButtonElement;
             if (dossierButton)
                 dossierButton.click();
+            else
+                notyf.error("Could not add nation to dossier.")
```

Comment:
This one's a bit unspecific (contrary to the other ones, which tell the user what's up). This condition would usually happen if the nation is already in the dossier.